solutions/usaco-1083.mdx

```diff
@@ -33,4 +33,29 @@ print(num_times)
 
 </PySection>
 
+<JavaSection>
+```java
+import java.io.*;
+
+public class UdderedButNotHerd {
+	public static void main(String[] args) throws IOException {
+		BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
+		String cowphabet = input.readLine();
+		char[] word = input.readLine().toCharArray();
+
+		int numTimes = 1;  // Every word must be hummed at least once.
```

Comment:
```suggestion
		int numTimes = 1;  // There must be at least one humming sequence.
```
idk feel like this is worded better

Reply:
imo original is fine

Reply:
i mean what does "word" mean?

Reply:
uhhh
the string of lowercase letters that farmer john heard bessie say.